test/e2e/install/install_test.go

```diff
@@ -90,12 +92,25 @@ var _ = Describe("[Install] continuousIntegration", func() {
 			}, 5*time.Minute, 10*time.Second).Should(BeTrue(), "RTE condition did not become available")
 
 			By("checking the NRT CRD is deployed")
-			crd := &apiextensionv1.CustomResourceDefinition{}
-			key := client.ObjectKey{
-				Name: crdName,
-			}
-			err := e2eclient.Client.Get(context.TODO(), key, crd)
+			_, err := crds.GetByName(e2eclient.Client, crdNRTName)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("checking the NRO CRD is deployed")
+			_, err = crds.GetByName(e2eclient.Client, crdNROName)
 			Expect(err).NotTo(HaveOccurred())
+
+			By("checking Daemonset is up&running")
+			workerNodes, err := getWorkerNodes()
+			Expect(err).NotTo(HaveOccurred())
+
+			ds, err := getDaemonSetByOwnerReference(updatedNROObj.UID)
+			Expect(err).NotTo(HaveOccurred())
+
+			workerNodesNumber := int32(len(workerNodes))
+			Expect(ds.Status.DesiredNumberScheduled).To(Equal(workerNodesNumber))
+			Expect(ds.Status.CurrentNumberScheduled).To(Equal(workerNodesNumber))
+			Expect(ds.Status.NumberReady).To(Equal(workerNodesNumber))
```

Comment:
this works in the happy path: lines 106-112 should be wrapped in a `Eventually` block.

Reply:
done